modules/openapi-generator/src/main/resources/typescript/http/servers.mustache

```diff
@@ -55,3 +51,7 @@ export class ServerConfiguration<T> {
 {{#servers}}
 export const server{{-index}} = new ServerConfiguration<{ {{#variables}} "{{name}}": {{#enumValues}}"{{.}}"{{^-last}} | {{/-last}}{{/enumValues}}{{^enumValues}}string{{/enumValues}}{{^-last}},{{/-last}} {{/variables}} }>("{{url}}", { {{#variables}} "{{name}}": "{{defaultValue}}" {{^-last}},{{/-last}}{{/variables}} })
 {{/servers}}
+
+{{#useInversify}}
+export const servers = [{{#servers}}server{{-index}}{{^-last}}, {{/-last}}{{/servers}}];
```

Comment:
Is this something we can just provide in any generator? Doesn't seem to be inversify specific.

Reply:
Sure, if you think it is useful, I will add it.

Reply:
Always including it now.